src/simulator_core/entities/assets/esdl_asset_object.py

```diff
@@ -1,11 +1,46 @@
 """Module containing classes to be able to interact with esdl objects."""
 import logging
+from dataclasses import dataclass
+
 from esdl import esdl
-from simulator_core.entities.assets import AssetAbstract, DemandCluster, ProductionCluster, Pipe
+
+from simulator_core.adapter.transforms.string_to_esdl import StringEsdlAssetMapper
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class EsdlKey:
+    """Class to hold the name and default value of an esdl key.
+
+    :param name: str the name of the key
+    :param default: any the default value of the key
+    """
+
+    name: str
+    default: any
+
+
+ASSET_DICT = {
+    "producer": {
+        "heating demand": EsdlKey(name="power", default=0.0),
+    },
+    "consumer": {
+        "heating demand": EsdlKey(name="power", default=0.0),
+        "maximum temperature": EsdlKey(name="maximumTemperature", default=0.0),
+        "minimum temperature": EsdlKey(name="minimumTemperature", default=0.0),
+    },
+    "pipe": {
```

Comment:
Hier zit nog een listigheid bij in ESDL kun je ook de DN specificeren zonder verder de Inner en Outer diameter op te geven. We moeten er ook over denken hoe de informatie van de isolatie dikte en materiaal eruit te halen

Reply:
Daar moeten we misschien nog even een method voor schrijven. Willen we dit in deze code doen of wordt dat een algemene methode? 

Ik kon de isolatie dikte en het materiaal ook niet direct vinden.